setup.py

```diff
@@ -612,7 +612,7 @@ def run(self):
     ],
     python_requires=">=3.8",
     install_requires=[
-        "scikit-learn>=1.0",
+        "scikit-learn>=1.1",
```

Comment:
This is used only in opensouce build and does not impact binary. also it might worth to align this with minimally actually supported version - i.e. 0.24 now